contracts/TalentLayerService.sol

```diff
@@ -109,6 +133,15 @@ contract TalentLayerService is Initializable, ERC2771RecipientUpgradeable, UUPSU
      */
     event ServiceDetailedUpdated(uint256 indexed id, string dataUri);
 
+    /**
+     * @notice Emit when data is updated for a Service
+     * @param id The service ID
+     * @param dataUri New service Data URI
+     * @param referralAmount New referral amount
+     * @param token New service token
+     */
+    event ServiceUpdated(uint256 indexed id, string dataUri, uint256 referralAmount, address token);
```

Comment:
- token should not be updatable 
- rename event to ServiceReferralAmountUpdated